### DISCUSSION TITLE
Add "German" (And More) To New Language Switcher

### DISCUSSION THREAD

**Discussion by djohannes** [ChristopherKlay/StadiaEnhanced#128] [Ideas & Requests]
The new language switcher is really cool, but "German" and others are unfortunately missing. 3 official Stadia countries use German (Germany, Austria, Switzerland), so I would really like to see "German" also for the language switcher. If you go by the supported countries, those would be missing too:

- Italian
- Swedish
- Danish
- Norwegian
- Finnish
- Dutch
- Polish
- Portuguese
- Czech
- Slovak
- Romanian
- Hungarian

**Comment by ChristopherKlay** [comment]
The language switcher wasn't targeted at supporting *all* possible languages, but mainly to allow people to easily switch games into a different language (mainly english) which is super useful when looking up guides and such for them.

If you live in germany (or better said, your account language is set to german or another german speaking country), Stadia will be german by default - you would only need to switch if you want to actually start a game in a *non-german* language. The "Default" option at the top always changes back to the language set in your Google account (german in this case) afterwards.

I do consider adding all/most available languages, their flags and other things down the line when Google pushes country support however and will switch this issue into a feature suggestion topic, as well as adding the changes to the project board (:

**Comment by Primajin** [comment]
Maybe it would be cool to select your three "favorite" languages from the Stadia enhanced settings menu so that you can for example set "English, German and Italian" as your preferred ones and then these three show up in the 🌐 menu - that way we also don't need to think about solving having a drop down with 25 different languages.
